components/generic/SmartLink.vue

```diff
@@ -45,7 +54,17 @@
           });
         }
         return this.destination;
+      },
+      isExternalLink() {
+        return parseDomain(this.destination);
```

Comment:
this won't be enough. We'll need to check that the domain is included in a list. Either it needs to match the Domain of the currently loaded page, or more likely we'll want to be able to set this via the .env file, something like:
`INTERTNAL_DOMAINS="europeana.eu, eanadev.org"`
Then we can set it for the various deployements.

Reply:
The only issue I would see with a whitelist is that content editors wouldn't be able to update it if they were to introduce a new external link

Reply:
As discussed elsewhere, it should just use the current domain.